### PR TITLE
+ bash completion

### PR DESCRIPTION
**Pull request by brokenpip3**

### CODE REVIEW

**Comment by schu** [comment]
Thanks! A few minor comments:

* Please move the file to `contrib/bash_completion.d/kubedee`.
* Rather write `Add bash completion` or `Add contrib/bash_completion.d/kubedee` as commit message.
* Can you squash the second commit into the first, as it's actually a fixup?

File: _kubedee [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    commands="controller-ip create create-admin-sa create-user-sa delete etcd-env kubectl-env list smoke-test start start-worker up version --help"
+
+    if [ $COMP_CWORD == 1 ]
+    then
+      COMPREPLY=($(compgen -W "${commands}" -- ${cur}))
+      return 0
+    fi
+
+	if [ $COMP_CWORD == 2 ]
```
Comment (schu) [inline _kubedee]: Indentation seems to be off.

File: _kubedee [hunk 2]
```diff
@@ -0,0 +1,69 @@
+_kubedee() {
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    commands="controller-ip create create-admin-sa create-user-sa delete etcd-env kubectl-env list smoke-test start start-worker up version --help"
```
Comment (schu) [inline _kubedee]: I think the command suggestion for help should be `help`, not the flag `--help`.

**Comment by schu** [comment]
Looks good to me modulo comments!

**Comment by brokenpip3** [comment]
I think I'm done, please tell me if there is something else you want change.

Ps this night/tomorrow I'm going to wrote a aur pkgbuild (arch linux user repo) for kubedee (versioning with last git commit)

**Pull request merged**